### DISCUSSION TITLE
Why Different, I can't find Create Domain

### DISCUSSION THREAD

**Discussion by beryindo** [ngoduykhanh/PowerDNS-Admin#1411] [Q&A] [answered]
Why Different, I can't find Create Domain ?

![Image](https://img001.prntscr.com/file/img001/7lvVzL1LSvugsd_0kVBQyA.jpg)

**Comment by metrax** [comment, marked as answer]
Please have a look to the Menu
![image](https://user-images.githubusercontent.com/1061866/221792648-9419ea36-3e7b-4d18-900f-2eb74e92f227.png)

**Comment by AzorianMatt** [comment]
@beryindo I recently started changing the chosen language to use the appropriate nomenclature of the technology as opposed to the previously used slang references.

**Comment by hapctic** [comment]
I was looking for the reason why my interface is different from the readme image. :)
It would be a good idea for dashboard areas to be movable blocks. the main information, which is the zone table, is currently the last element.
Thanks for the work.

**Comment by AzorianMatt** [comment]
@hapctic There won't be a need as I'm going to eliminate the other blocks from the dashboard anyhow since they are fairly useless in practice.

**Reply by AzorianMatt** [reply]
@hapctic Those additional blocks have been eliminated in the current dev branch and this change will be part of the upcoming 0.4.0 release this month.
